### ISSUE TITLE
Cached when failed.

### ISSUE THREAD

**Issue by SettingDust** [actionsflow/actionsflow#23]
## Description

The rss item is cached when the jobs failed.

### Steps to reproduce

Use rss trigger. Jobs failed. Cached

### Expected result

If failed. Don't cache

**Comment by theowenyoung** [comment]
Yes, currently, when RSS items were fetched success, then it'll be cached.

The workflow now is separated, actionsflow fetch the items, and act run actions. It's difficult to callback act's result to actionsflow.

Maybe there is a better way to do this, PR is welcome.
